fvtest/compilertriltest/LogicalTest.cpp

```diff
@@ -59,14 +63,74 @@ TEST_P(Int32LogicalUnary, UsingConst) {
     ASSERT_EQ(exp, act);
 }
 
-
 INSTANTIATE_TEST_CASE_P(LogicalTest, Int32LogicalUnary, ::testing::Combine(
     ::testing::ValuesIn(TRTest::const_values<int32_t>()),
     ::testing::Values(
         std::make_tuple("ineg", ineg),
         std::make_tuple("iabs", iabs)
         )));
 
+class Int64LogicalUnary : public TRTest::UnaryOpTest<int64_t> {};
+
+TEST_P(Int64LogicalUnary, UsingConst) {
+    auto param = TRTest::to_struct(GetParam());
+
+    char inputTrees[120] = {0};
+    std::snprintf(inputTrees, 120,
+        "(method return=Int64
```

Comment:
These kinds of multi-line strings are not allowed in C++. Each line needs to start and end with `"`